### PR TITLE
German Language de.json

### PR DESCRIPTION
**Pull request by osworx**
Localized German strings - enjoy and good luck with the project.

### CODE REVIEW

File: src/resources/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "modal_trigger_title": "Cookieeinstellungen",
+
+    "consent_modal_title": "Wir verwenden Cookies ..",
```
Comment (jelen07) [inline src/resources/translations/de.json:4]:
```suggestion
    "consent_modal_title": "Wir verwenden Cookies",
```

**Comment by jelen07** [comment]
Hi @osworx, thank you for the translation 👍 There are a few little things.

File: src/resources/translations/en.json [hunk 2]
```diff
@@ -1,36 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (osworx) [reply src/resources/translations/en.json:1]: Yes of course - have I deleted it, sorry!

**Comment by jelen07** [comment]
Duplicity with https://github.com/68publishers/cookie-consent/pull/45, closing.

**Pull request closed without merging**